psi4/driver/inputparser.py

```diff
@@ -693,8 +693,8 @@ def process_input(raw_input, print_level=1):
     #    check_parentheses_and_brackets(temp, 1)
 
     # First, remove everything from lines containing only spaces
-    blankline = re.compile(r'^\s*$')
-    temp = re.sub(blankline, '', temp, re.MULTILINE)
+    blankline = re.compile(r'^\s*$', re.MULTILINE)
+    temp = re.sub(blankline, '', temp)
```

Comment:
Same as below, the function is `re.sub(pattern, repl, string, count=0, flags=0)`. So this should work, unless I misunderstand something. I do not use advanced regex much so I am likely overlooking something.